.nuspec/Xamarin.Forms.targets

```diff
@@ -18,7 +26,7 @@
 	<PropertyGroup>
 		<_XFTargetsImportedAgain Condition="'$(_XFTargetsImported)'=='True'">True</_XFTargetsImportedAgain>
 		<_XFTargetsImported>True</_XFTargetsImported>
-		<IsXBuild Condition="'$(MSBuildRuntimeVersion)' == ''">true</IsXBuild>
+		<IsXBuild Condition="'$(MSBuildRuntimeVersion)' == '' and '$(MSBuildRuntimeType)' != 'Core'">true</IsXBuild>
```

Comment:
if the left part of the comparison is `true`, so is the second. you might mean `or`